scripts/verify.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+make -s -C ..
```

Comment:
Don't execute `make` here. Instead, use `which` and shell scripts to validate the executable file.